database/src/main/java/com/pip/chatbot/repository/food/FoodRepository.java

```diff
@@ -49,21 +52,26 @@
                 .into(Restaurant.class));
     }
 
-    public Optional<Mark> createMark(Mark mark){
-        var result = dsl.insertInto(MARK_FOOD)
-                .set(MARK_FOOD.RESTAURANT_ID, mark.getRestaurantId())
-                .set(MARK_FOOD.MARK, mark.getMark())
+    public Optional<Mark> createMark(Mark mark) {
+        var result = dsl.insertInto(MARK_RESTAURANT)
+                .set(MARK_RESTAURANT.RESTAURANT_ID, mark.getRestaurantId())
+                .set(MARK_RESTAURANT.MARK, mark.getMark())
                 .returning()
                 .fetchOne();
 
         return Optional.ofNullable(result.into(Mark.class));
     }
 
     public Optional<Mark> getAvgRestaurantMark(String id) {
-        var result = dsl.select(MARK_FOOD.RESTAURANT_ID, avg(MARK_FOOD.MARK).as("mark"))
-                .from(MARK_FOOD)
-                .where(MARK_FOOD.RESTAURANT_ID.eq(Integer.parseInt(id)))
-                .groupBy(MARK_FOOD.RESTAURANT_ID)
+        var avgMark = dsl.select(MARK_RESTAURANT.RESTAURANT_ID, avg(MARK_RESTAURANT.MARK).as("mark"))
```

Comment:
To zróbmy jednym zapytaniem wybierając ocenę z zomato plus średnią z naszych userów. Jednego joina trzeba zrobić:)